packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -452,6 +454,10 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
         payable
         metered(_gasLimit)
     {
+        // This function locks ETH in the SharedLockbox when using the OptimismPortalInterop contract.
+        // If the interop version is not used, this function is a no-ops.
+        if (msg.value != 0) _lockETH();
```

Comment:
Can we put the `if (msg.value != 0)` inside the implementation of `_lockETH()` so it is a noop? There is new logic going into this contract, the branch will always exist